Configuration/TypoScript/Setup/Part4GlobalConfiguration.typoscript

```diff
@@ -0,0 +1,29 @@
+// Part 4: global site configuration
+config {
+  absRefPrefix = auto
```

Comment:
Should we output those default values here or go for a "minimum" config?
IMHO that makes it easier to keep the extension "best practice", even if Core default values change (or get obsolete).

Reply:
I think I would prefer going for minimum. However I am not really sure what is a best-practise here. Can you make a suggestion? I just kept the config values as they were for LTS 10...

Reply:
I've checked them:

  absRefPrefix = auto # default is '' - I am not sure about that one; I would say the default is fine
  cache_period = 86400 # is default
  debug = 0 # if left out it will assume the value of $GLOBALS['TYPO3_CONF_VARS']['FE']['debug'] - which would be better IMHO
  disablePrefixComment = 1 # default is 0; not sure because it actually does add unnecessary TypoScript rendering debug output - anyways it does also not hurt
  doctype = html5 # default
  extTarget = # default
  index_enable = 1 # indexed_search: should be not configured here IMHO
  index_externals = 1 # indexed_search: should be not configured here IMHO
  index_metatags = 1 # indexed_search: should be not configured here IMHO
  inlineStyle2TempFile = 1 # default
  intTarget = # default
  linkVars = L # probably obsolete or at least not needed
  metaCharset = utf-8 # default
  no_cache = 0 # default
  pageTitleFirst = 1 # too specific here IMHO
  prefixLocalAnchors = all # too specific here IMHO
  removeDefaultJS = 0 # default: 'external'; default is just fine IMHO
  renderCharset = utf-8 # deprecated since 8.0
  sendCacheHeaders = 1 # default: 0; I would rather not configure it here and roll with the TYPO3 default
  uniqueLinkVars = 1 # removed in 7.0

  // Compression and concatenation of CSS and JS Files
  compressCss = 0 # default
  compressJs = 0 # default
  concatenateCss = 0 # default
  concatenateJs = 0 # default